### PR TITLE
Fix disconnected iSCSI SR (error 202)

### PR DESCRIPTION
**Pull request by benjamreis**
Reverts commit d28dcc1341166ed4839dbea8ef6d1a24e981f461.

### CODE REVIEW

**Comment by benjamreis** [comment]
Waiting for https://github.com/xcp-ng/sm/pull/63

File: SPECS/sm.spec [hunk 1]
```diff
@@ -11,7 +11,7 @@
 Summary: sm - XCP storage managers
 Name:    sm
 Version: 2.30.8
-Release: %{?xsrel}.1%{?dist}
+Release: %{?xsrel}.1.0.fix202.1%{?dist}
```
Comment (Wescoeur) [inline SPECS/sm.spec:14]: Why this tag? We will release this change for everyone.
No difference with the previous release, it's just a new patch.

**Comment by benjamreis** [comment]
[v8.2-incoming scratch build](https://koji.xcp-ng.org/taskinfo?taskID=61737)

**Comment by stormi** [comment]
This is an important change, so why it is necessary, what exactly is the problem it fixes, and why this is the best solution for now, as well as whether it's a temporary or permanent fix, should all be documented somewhere visible from this commit. Probably in the https://github.com/xcp-ng/sm commit, so that it's included in the patch header.

**Review by stormi — APPROVED** [review]

**Review by Wescoeur — APPROVED** [review]

**Comment by benjamreis** [comment]
[v8.2-incoming build](https://koji.xcp-ng.org/taskinfo?taskID=61974)

**Pull request merged**